resources/views/index.blade.php

```diff
@@ -4,7 +4,7 @@
     @include('layouts.sidebar', ['sidebar'=> Menu::get('sidebar_admin')])
 @endsection
 @section('css')
-    <link rel="stylesheet" href="{{mix('/css/package.css', 'vendor/processmaker/packages/package-skeleton')}}">
+{{--    <link rel="stylesheet" href="{{mix('/css/package.css', 'vendor/processmaker/packages/package-skeleton')}}">--}}
```

Comment:
Consider to remove this commented code.
If want to add it as example of how to add an css to a file, consider to place it as example in the REAME.md